app/upgrades/v1_7.go

```diff
@@ -166,6 +166,40 @@ func V010706UpgradeHandler(
 	appKeepers *keepers.AppKeepers,
 ) upgradetypes.UpgradeHandler {
 	return func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		if isMainnet(ctx) || isTest(ctx) {
+			// unblock 2x stuck unbondings
+			hashes := []struct {
+				Zone   string
+				Hash   string
+				Status int32
+			}{
+				{Zone: "regen-1", Hash: "ee0b5f5c423508c8dd6a501168a77a0b72d5a8aaf1702a64804e522334ff272b", Status: icstypes.WithdrawStatusUnbond},
```

Comment:
Not sure about where these hashes are coming from. But if already verified then good to go here